app/javascript/components/search/genes/GeneSearchView.js

```diff
@@ -56,22 +64,27 @@ export default function GeneSearchView() {
   return (
     <div>
       <div className="row">
-        <div className="col-md-6 col-sm-12 col-xs-12">
-          <form onSubmit={ handleSubmit }>
+        <div className="col-md-12 col-sm-12 col-xs-12">
+          <form className="gene-keyword-search form-horizontal" onSubmit={ handleSubmit }>
             <div className="input-group">
               <input type="text"
-                className="form-control gene-search-input"
+                className="form-control"
                 value={genes}
+                size="50"
                 onChange={ e => setGenes(e.target.value) }
-                placeholder={ geneSearchPlaceholder }/>
-              <div className="input-group-btn">
-                <button className="btn btn-info"
-                        type="submit"
-                        name="commit"
-                        id="submit-gene-search">
-                  <span className="fas fa-search"></span>
-                </button>
+                placeholder={ geneSearchPlaceholder }
+                name="genesText"/>
```

Comment:
We've been using kebab-case for attribute values, so for consistency:

```suggestion
                name="genes-text"/>
```

And similarly for other references to this attribute, e.g. in tests.